core/src/main/java/org/web3j/protocol/websocket/WebSocketSubscription.java

```diff
@@ -0,0 +1,21 @@
+package org.web3j.protocol.websocket;
+
+import rx.subjects.BehaviorSubject;
+
+public class WebSocketSubscription<T> {
```

Comment:
Please can you ensure all classes have a brief Javadoc description - a single sentence is fine.

Reply:
Done.